extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -137,24 +137,24 @@ describes.realWin('amp-list component', {
     });
 
     it('should fetch and render', () => {
-      const items = [{title: 'Title1'}];
+      const fetched = {'items': [{title: 'Title1'}]};
```

Comment:
I think this change is a function of how the test works, and does not imply a change in code logic, but a second opinion here would be helpful.

Reply:
seems fine to me. I can see how this is the result of `fetch_` not taking `expr` anymore and the implications of it with the mocking. Also looks "more correct" than the old version.

Reply:
Thanks for the sanity check! I pulled this out to a constant so that in case we ever have to change this again, we don't have to type it 30 times in this file.